bedrock/mozorg/templates/mozorg/diversity/2021/beyond-products.html

```diff
@@ -0,0 +1,483 @@
+          image_url='img/mozorg/diversity/2021/card-images/auc-data-science.jpg',
+          )}}
+        </div>
+      </div>
+    </div>
+  </section>
+  <!-- Modal content -->
+  <div class="mzp-l-content js-modal-content">
+    <!-- Common voice -->
+    <article data-modal-parent="common-voice" id="common-voice" class="mzp-c-card-feature mzp-has-aspect-16-9 mzp-l-card-feature-left-half">
+      <div class="mzp-c-card-feature-media-wrapper">
+        <div class="mzp-c-card-feature-media">
+          {{ image(url='img/mozorg/diversity/2021/card-images/mozilla-common-voice.jpg', loading='lazy', width='930', height='523', class='mzp-c-card-image') }}
+        </div>
+      </div>
+      <div class="mzp-c-card-feature-content">
+        <div class="mzp-c-card-feature-content-container">
+          <img src="{{ static('img/foundation/annualreport/2019/modal-dots-2.svg') }}" alt="">
+          <h2 class="mzp-c-card-feature-title"><span class="highlight violet">Mozilla Common Voice</span></h2>
+          <p class="mzp-c-card-feature-desc">Over the next decade, speech is expected to become the primary way people interact with devices — from phones and laptops to digital assistants. Today’s voice-enabled devices, however, are inaccessible to vast swaths of the planet’s languages, accents, and speech patterns. Mozilla Common Voice launched in 2017 to bridge this gap and is now the world’s largest multi-language public domain voice data set, with over 18,000 hours of voice recordings in over 100 global languages and dialects. Common Voice enables anyone to donate their voices to a free publicly available database that startups, researchers, and developers can use to train voice-enabled apps, products, and services. In 2021, The Bill & Melinda Gates Foundation, the Deutsche Gesellschaft für Internationale Zusammenarbeit (GIZ) GmbH (German Development Cooperation), and the UK’s Foreign, Commonwealth & Development Office (FCDO) announced an investment of $3.4 million in Mozilla Common Voice. This funding will support the collection of voice data sets in Kiswahili, an East African language spoken by an estimated 100 million people in Kenya, Uganda, Tanzania, Rwanda, Burundi, and South Sudan.</p>
+          <a target="_blank" rel="external noopener" class="mzp-c-button mzp-t-secondary" href="https://foundation.mozilla.org/en/blog/mozilla-common-voice-receives-34-million-investment-to-democratize-and-diversify-voice-tech-in-east-africa/">Learn more</a>
+        </div>
+      </div>
+    </article>
+    <!-- AI for humanity -->
+    <article data-modal-parent="ai-humanity" id="ai-humanity" class="mzp-c-card-feature mzp-has-aspect-16-9 mzp-l-card-feature-left-half">
+      <div class="mzp-c-card-feature-media-wrapper">
+        <div class="mzp-c-card-feature-media">
+          {{ image(url='img/mozorg/diversity/2021/card-images/weforum.jpg', loading='lazy', width='930', height='523', class='mzp-c-card-image') }}
+        </div>
```

Comment:
This goes for most images, but they overall seem like they are much bigger than they are displaying on the pages - this one is displaying at 287 x 161 but the image is sized at 1200 x 675 - you could probably save on image sized if these were brought down in size?